clinica/cmdline.py

```diff
@@ -62,7 +71,7 @@ def setup_logging(verbose: bool = False) -> None:
         {
             "logging": {"log_directory": os.getcwd(), "log_to_file": True},
             "execution": {"check_version": False},
-        },
+        }
```

Comment:
Unless I am missing something, the nipype logger will not be affected and still write to the current folder, right ?

Reply:
I believe so. There is nothing in nipype's [config code](https://github.com/nipy/nipype/blob/master/nipype/utils/config.py) that suggests otherwise.